experiments/test_tony1.py

```
@@ -0,0 +1,61 @@
+import subprocess
+import tempfile
+import shutil
+from pathlib import Path
+
+def run_exiftool_command(cmd_args, media_path):
+    cmd = ["exiftool", "-overwrite_original", "-charset", "filename=UTF8", str(media_path)]
+    cmd.extend(cmd_args)
+    subprocess.run(cmd, capture_output=True, text=True, check=True, timeout=30, encoding='utf-8')
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)